provider/resource.go

```diff
@@ -173,6 +177,11 @@ func (r *resource) TFResource() *schema.Resource {
 	return r.tfResource
 }
 
+func (r *resource) SetImporter(i *schema.ResourceImporter) {
+	resource := r.TFResource()
```

Comment:
```suggestion
	r.TFResource().Importer = i
```
`r.TFResource()` will panic if there is an issue, it does not return `error`, so we can oneline it :)